quipucords/api/messages.py

```diff
@@ -108,6 +108,17 @@
 SJ_EXTRA_VARS_KEY = 'Extra vars keys must be jboss_eap, jboss_fuse, '\
     'or jboss_brms.'
 
+SJ_MERGE_JOB_REQUIRED = 'This field is required'
+SJ_MERGE_JOB_NOT_LIST = 'This field must be a list of job ids.'
+SJ_MERGE_JOB_TOO_SHORT = 'Two or more job ids are required.'
+SJ_MERGE_JOB_NOT_INT = 'Job ids must be integers.'
+SJ_MERGE_JOB_NOT_UNIQUE = 'Set of ids must be unique.'
+SJ_MERGE_JOB_NOT_FOUND = 'Not all job ids exist. Jobs note found: %s'
```

Comment:
Jobs note found --> jobs not found

Reply:
let's use scan job just to be specific.... everywhere!

Reply:
Done for both